### PR TITLE
fix: button component add colorErrorBgActive token

### PR DESCRIPTION
**Pull request by LonelySnowman**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

-  close：https://github.com/ant-design/ant-design/issues/46288

### 💡 Background and solution

Button component add color error background active token

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Button component add color error background active token |
| 🇨🇳 Chinese | 按钮组件新增错误激活态浅色背景 token |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/theme/themes/shared/genColorMapToken.ts [hunk 1]
```diff
@@ -60,6 +60,7 @@ export default function genColorMapToken(
 
     colorErrorBg: errorColors[1],
     colorErrorBgHover: errorColors[2],
+    colorErrorBgActive: errorColors[3],
```
Comment (afc163) [inline components/theme/themes/shared/genColorMapToken.ts:63]: 直接用 colorErrorActive 呢？

Comment (LonelySnowman) [reply components/theme/themes/shared/genColorMapToken.ts:63]: @afc163 colorErrorActive 是深色的激活态，是不是色差有点大。

image
image

Comment (afc163) [reply components/theme/themes/shared/genColorMapToken.ts:63]: 确实

**Review by afc163 — APPROVED** [review]

**Comment by li-jia-nan** [comment]
加新 token 不需要提交到 feature 分支吗？

**Pull request merged**